### DISCUSSION TITLE
.net 4.8 compatibility

### DISCUSSION THREAD

**Discussion by elemariosClearstar** [mganss/HtmlSanitizer#392] [Q&A]
I dowloaded the package in a project .net 4.8 , it locally compiles and works fine....but remotely I get "The type initializer for Ganss.XSS.HtmlSanitizer threw an exception" , I'm unable to see the inner exception. Any clue on what is happening? ( I notice that the requirement is "=" .net  4.6 , maybe is not fully compatible )

**Comment by mganss** [comment]
HtmlSanitizer supports .NET Framework 4.8 (which implements [.NET Standard 2.0](https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0)). What do you mean by "remotely"? Perhaps there is a version mismatch between the two environments you have tested?

**Reply by elemariosClearstar** [reply]
Thanks for the reply. By remotely I mean the deployed App. I have an update on this, I was able to get the exception details and the problem was some dependencies missing, it seems I accidentally deleted them but my local continue working with some cached ones  . It is now fixed. Thank you
